apps/account/tests/test_models.py

```
@@ -1,146 +1,162 @@
 
-        self.assertEqual(qs.count(), 2)
+    assert qs.count() == 1
+    assert qs.first()["name"] == room.name
 
-    def test_has_seen_room_true(self):
-        connection = UserConnectionToRoom.objects.get(user=self.user)
-        connection.user_has_seen_this_room = True
-        connection.save()
 
-        found_connection, has_seen_room = self.user.has_seen_room(self.room.id)
+def test_room_qs_for_list_superuser(superuser, room):
```

Ruff (0.14.7)

Unused function argument: `room`

(ARG001)